### PR TITLE
Flash messages

### LINKED ISSUE #529: Blue alerts/flash notifications are slightly off vertical centre.
Is it possible to move these down a little so they are centred? Not sure if this is a design system issue or pursuit, so putting it here. 

screenshot 2016-02-02 12 02 45

### PR DESCRIPTION
**Pull request by axelboc**
Fixes #529. Brings notices to the top of the page to display, for instance, the result of a form submission (when Ajax or JS is not available). This will be used for the subscriptions on Pursuit.

### CODE REVIEW

**Comment by axelboc** [comment]
The second commit provides some usage examples for testing. It needs to be reverted before merging.

File: assets/targets/components/notices/_flash.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  .flash--warning {
+    @extend %notice-warning;
+  }
+
+  .flash--danger {
+    @extend %notice-danger;
+  }
+
+  .flash--centre {
```
Comment (waitingallday) [inline assets/targets/components/notices/_flash.scss]: I hate to bring it up, but can this be `.flash--center` for consistency with css? I would prefer the correct spelling, but I know I'll typo this repeatedly!

Comment (axelboc) [inline assets/targets/components/notices/_flash.scss]: Haha sure, I'll change it. There's a class called `center`, so it makes sense for consistency as well.

Comment (waitingallday) [inline assets/targets/components/notices/_flash.scss]: :sob:

**Comment by axelboc** [comment]
Were you happy leaving the usage examples in the layouts?

**Comment by waitingallday** [comment]
Yeah absolutely, I think it helps to see it in context :+1:

**Pull request merged**